postgraas_server/management_resources.py

```diff
@@ -71,12 +71,12 @@ def get(self, id):
 
     def delete(self, id):
         parser = reqparse.RequestParser()
-        parser.add_argument('db_pwd', required=True, type=str, help='pass of the db user')
+        parser.add_argument('db_pwd', required=True, type=str, help='pass of the db user is needed to delete instance.')
         args = parser.parse_args()
 
         entity = DBInstance.query.get(id)
         if not entity:
-            return {'status': 'failed', 'msg': 'Postgraas instance {} does not exist'.format(id)}
+            return {'status': 'failed', 'msg': 'Postgraas instance {} does not exist'.format(id)}, 404
```

Comment:
ACK, strange, that they mix it up here, too:
http://flask-restful.readthedocs.io/en/latest/quickstart.html#full-example

 abort(404, message="Todo {} doesn't exist".format(todo_id))

vs

return '', 204

Reply:
The difference is that `abort` raises an exception rather than simply returning something. It can therefore happen deep inside the call stack.

In the given context it does not really matter what we use. However, given that we already use `abort` for other error messages, I agree that `abort` will be more consistent.

Reply:
I guess it is ok in case of success (=2XX) to have `return result, 201`, and in all kinds of failure `abort(500, ...). Just having some failures with `return result, 400` and others with abort is strange.